app/controllers/status_controller.rb

```diff
@@ -1,9 +1,22 @@
+# frozen_string_literal: true
```

Comment:
Not blocking: out of curiosity, what is the purpose of implicitly freezing all string literals in this file? I don't see right off the bat why we want to do this.

Reply:
basically to stop string allocations for static string data in code, checkout this https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/

Reply:
Oooo very nice!